modules/labs-react/common/lib/Box.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import isPropValid from '@emotion/is-prop-valid';
+import {createComponent, StyledType} from '@workday/canvas-kit-react/common';
+
+// style props
+import {border, BorderProps} from './utils/border';
+import {color, ColorProps} from './utils/color';
+import {depth, DepthProps} from './utils/depth';
+import {flexItem, FlexItemProps} from './utils/flexItem';
+import {layout, LayoutProps} from './utils/layout';
+import {position, PositionProps} from './utils/position';
+import {space, SpaceProps} from './utils/space';
+
+type StyleProps = BorderProps &
+  ColorProps &
+  DepthProps &
+  FlexItemProps &
+  LayoutProps &
+  PositionProps &
+  SpaceProps;
+
+export type BoxProps = StyledType &
```

Comment:
`StyledType` should not be a part of `BoxProps`. It should be a part of `StyledBox`